eclipse/plugins/org.switchyard.tools.ui.bpmn2/META-INF/MANIFEST.MF

```diff
@@ -21,9 +21,9 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.emf.edit.ui,
  org.eclipse.emf.transaction,
  org.eclipse.bpmn2,
- org.eclipse.bpmn2.modeler.core;bundle-version="[1.1.0,1.4.0)",
- org.eclipse.bpmn2.modeler.ui;bundle-version="[1.1.0,1.4.0)",
- org.eclipse.bpmn2.modeler.runtime.jboss.jbpm;bundle-version="[1.2.3,1.4.0)",
+ org.eclipse.bpmn2.modeler.core;bundle-version="[1.1.0,1.5.0)",
```

Comment:
Usually a particular version is only going to work with a particular release of Eclipse, so it's more to keep in line with the version range we know will support Oxygen or Neon or whatever is next I suspect. @pleacu do you have a better answer?